app/policies/evaluation_policy.rb

```diff
@@ -34,6 +50,10 @@ def leader_of_the_resort?
     user.leader_of?(evaluation.group.parent)
   end
 
+  def pek_admin?
```

Comment:
Can we move this to a global policy (maybe application policy) or something similar? Sounds like we want to use this in several policy.

Reply:
Moved this and some other methods to ApplicationPolicy.